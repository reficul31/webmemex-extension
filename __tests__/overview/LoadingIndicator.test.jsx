import React from 'react'
import LoadingIndicator from 'src/overview/components/LoadingIndicator'
import renderer from 'react-test-renderer'

describe('LoadingIndicator snapshot tests', () => {
    test('should test against the given snapshot', () => {
        const component = renderer.create(
            <LoadingIndicator />
            )
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
