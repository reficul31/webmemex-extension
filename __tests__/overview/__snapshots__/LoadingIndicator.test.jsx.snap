// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator snapshot tests should test against the given snapshot 1`] = `
<div
  className="container"
>
  <span
    className="dotone dot"
  />
  <span
    className="dottwo dot"
  />
  <span
    className="dotthree dot"
  />
</div>
`;
